From 420aaefee0f4adb4b5e2ab5f5930d50f2e6d83bc Mon Sep 17 00:00:00 2001
From: Matt Gumbel <matthew.k.gumbel@intel.com>
Date: Thu, 7 Nov 2013 13:02:23 -0800
Subject: Allow sdk and atf make targets at same time

AOSP disallows other device build targets at the same time as an SDK
build (for unknown reasons). However, ATF is special in that we want to
be able to build it right along with the SDK for CI in ABT.

Category: AOSP Enhancement
Upstream-Candidate: No
Issue: ADEV-269
Change-Id: Iee5d8fc1903a7baa51e16d66389ce9193402616e
Signed-off-by: Matt Gumbel <matthew.k.gumbel@intel.com>
---
 core/main.mk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/core/main.mk b/core/main.mk
index d424b25..d4a5f5c 100644
--- a/core/main.mk
+++ b/core/main.mk
@@ -360,7 +360,7 @@ ifdef is_sdk_build
 sdk_repo_goal := $(strip $(filter sdk_repo,$(MAKECMDGOALS)))
 MAKECMDGOALS := $(strip $(filter-out sdk_repo,$(MAKECMDGOALS)))
 
-ifneq ($(words $(filter-out $(INTERNAL_MODIFIER_TARGETS) checkbuild,$(MAKECMDGOALS))),1)
+ifneq ($(words $(filter-out $(INTERNAL_MODIFIER_TARGETS) atf checkbuild,$(MAKECMDGOALS))),1)
 $(error The 'sdk' target may not be specified with any other targets)
 endif
 
-- 
1.7.4.1

